shared/index.native.js

```diff
@@ -62,7 +63,7 @@ class Keybase extends Component {
     let child
 
     if (dumbSheetOnly) {
-      child = <DumbSheet />
+      child = <Box style={{flex: 1, marginTop: 40}}><DumbSheet /></Box>
```

Comment:
This was below the status bar. Just picked 40 as a big enough value to move it out of the way